File: Fibonacci.java
import java.util.*;
public class Fibonacci {
    public static void main(String[]args){
        Scanner input = new Scanner(System.in);
        System.out.print("Enter a num: ");
        int n = input.nextInt();
        int a = 0, b = 1, c;
        for(int i = 0; i < n; i++){
            System.out.println(a);
            c = a+b;
            a=b;
            b=c;
        }
    }
}

File: Fibonacci.c
#include <stdio.h>
void main(){
	printf("Enter a num: ");
	int n, a = 0, b = 1, i, c;

	scanf("%d",&n);
	for(i = 0; i < n; i++){
		printf("%d\n",a);
		c = a + b;
		a = b;
		b = c;
	
	}
}
